### ISSUE TITLE
Component: Chips: Invalid aria attribute: aria-pointset (should be aria-posinset)

### ISSUE THREAD

**Issue by barbGit2** [primefaces/primeng#14568]
### Describe the bug

The **aria-pointset** attribute should be replaced with **aria-posinset** in the **li.p-chips-token** elements of Chips.

### Environment

PrimeNG 16 and 17

### Angular version

16

### PrimeNG version

16 (and 17)

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

16.20.2

### Browser(s)

Chrome

### Steps to reproduce the behavior

See github codebase

### Expected behavior

Behavior is fine, but the Chips component will fail AXE tests with "ARIA attributes must conform to valid names (aria-valid-attr)" > "Invalid ARIA attribute name: aria-pointset"
